tests/modules/stanford_samlauth_test/stanford_samlauth_test.info.yml

```diff
@@ -0,0 +1,6 @@
+name: 'Test Stanford SSP'
+type: module
+core_version_requirement: ^8.8 || ^9
```

Comment:
Shouldn't this be set to `^9.5 || ^10`

Reply:
good eyes. it will help relieve any issue when we begin doing testing using D10